docs/framework/unmanaged-api/hosting/icorruntimehost-interface.md

```diff
@@ -71,6 +71,6 @@ Provides methods that enable the host to start and stop the common language runt
  <xref:System.AppDomain>   
  [Hosting](../../../../docs/framework/unmanaged-api/hosting/index.md)   
  [ICLRRuntimeHost Interface](../../../../docs/framework/unmanaged-api/hosting/iclrruntimehost-interface.md)   
- [Runtime Hosts](http://msdn.microsoft.com/en-us/99d9246a-b994-4fe5-985c-8588d1d59998)   
+ [Runtime Hosts](http://msdn.microsoft.com/99d9246a-b994-4fe5-985c-8588d1d59998)   
```

Comment:
ideally we'd add library to these links. otherwise customers can't get to versions that work. even better would be to link to a previous version that work but you can't tell by looking at the link